packages/renderer/src/lib/feedback/SendFeedback.svelte

```diff
@@ -43,131 +24,29 @@ function hideModal(): void {
   displayModal = false;
 
   // reset fields
-  smileyRating = 0;
-  tellUsWhyFeedback = '';
-  contactInformation = '';
   category = DEFAULT_CATEGORY;
 }
-
-function selectSmiley(item: number): void {
-  smileyRating = item;
-}
-
-async function sendFeedback(): Promise<void> {
-  const properties: FeedbackProperties = {
-    category: category,
-    rating: smileyRating,
-  };
-
-  if (tellUsWhyFeedback) {
-    properties.comment = tellUsWhyFeedback;
-  }
-
-  if (contactInformation) {
-    properties.contact = contactInformation;
-  }
-
-  // send
-  await window.sendFeedback(properties);
-  // close the modal
-  hideModal();
-}
-
-async function openGitHub(): Promise<void> {
-  hideModal();
-  await window.telemetryTrack('feedback.openGitHub');
-  await window.openExternal('https://github.com/containers/podman-desktop');
-}
 </script>
 
 {#if displayModal}
-  <Dialog title="Share your feedback" on:close={() => hideModal()}>
-    <svelte:fragment slot="content">
+  <Modal name="Share your feedback" on:close={hideModal}>
```

Comment:
question : why the switch from dialog to modal ?

Reply:
Because the Dialog template uses slots for each part (content, validation, and buttons), I wasn't able to find a way to both include the dropdown in `SendFeedback` and the `DevelopersFeedback` or `GitHubIssueFeedback` in this Dialog.
 `DevelopersFeedback` and `GitHubIssueFeedback` have different button with different functionalities and disabled situation, and different validations, so they cannot share those slots.
 The new `FeedbackForm.svelte` component uses the same slots introduced in Dialog, and  `DevelopersFeedback` and `GitHubIssueFeedback` use it.